### PR TITLE
Upgrade to latest raiden

### PR DESCRIPTION
**Pull request by palango**
After https://github.com/raiden-network/raiden/pull/5728

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
-git+https://github.com/raiden-network/raiden.git@f9bbbc0aeadf2ce1a54f3ed22de948133bc461df
-raiden-contracts==0.33.3
+git+https://github.com/palango/raiden.git@4dca1d2f35aaa94505cf82e3a2b890bf2106b7df
```
Comment (Dominik1999) [inline requirements.txt]: is the the correct link? It is not raiden-network but palango, now?

File: tests/libs/test_matrix.py [hunk 2]
```diff
@@ -112,12 +112,6 @@ def test_deserialize_messages_valid_messages(request_monitoring_message):
     assert len(messages) == 2
 
 
-def test_matrix_http_retry_delay():
```
Comment (Dominik1999) [inline tests/libs/test_matrix.py]: we don't need that test anymore?

File: requirements.txt (hunk 1 above)
Comment (palango) [reply requirements.txt]: This was the only way to test this before the PR in raiden is merged, I'll change that back. Just wanted CI to run already.

File: tests/libs/test_matrix.py (hunk 2 above)
Comment (palango) [reply tests/libs/test_matrix.py]: Fixed

File: src/raiden_libs/matrix.py [hunk 3]
```diff
@@ -120,8 +123,8 @@ def deserialize_messages(
 def matrix_http_retry_delay() -> Iterable[float]:
     return timeout_exponential_backoff(
         DEFAULT_TRANSPORT_RETRIES_BEFORE_BACKOFF,
-        int(DEFAULT_TRANSPORT_MATRIX_RETRY_INTERVAL / 5),
```
Comment (palango) [inline src/raiden_libs/matrix.py]: @karlb Any idea why `mypy` didn't catch this.

**Pull request merged**